.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Module",
+            "type": "python",
+            "request": "launch",
+            "module": "coverage",
+            "arguments": [
+                "run",
+                "--append",
+                "--source=admin_interface",
+                "setup.py",
+                "test"
+            ],
+            "justMyCode": true
+        }
+    ]
+}
```

Comment:
Did you commit this by accident?  FWIW I never use `git commit --add` or `git add .`, always `git add -p` + `git status` to avoid this kind of issue 🙂

Reply:
sorry. I'm always used to having it in my gitignore that I didn't a second thought. Adding it and `.venv` in the gitignore in the next commit